src/components/formik/devicePicker/index.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+
+import {
+  Dropdown,
+  IDropdownStyles,
+  IDropdownOption
+} from "office-ui-fabric-react/lib/Dropdown";
+
+const assetTypes: IDropdownOption[] = [
+  { key: "notebook", text: "Notebook" },
+  { key: "smartphone", text: "Smartphone" },
+  { key: "tablet", text: "Tablet" },
+  { key: "accessori", text: "accessori" }
+];
+
+const dropdownStyles: Partial<IDropdownStyles> = {
+  dropdown: { width: 250 }
+};
+
+const FormikDevicepicker = ({
+  name,
+  values,
+  label,
+  errors,
+  touched,
+  handleChange,
+  ...rest
+}) => {
+    console.log(values);
```

Comment:
Consider removing `console.log` or at least leave it with the proper indentation.